code/modules/research/xenobiology/crossbreeding/regenerative.dm

```diff
@@ -164,20 +174,23 @@ Regenerative extracts:
 
 /obj/item/slimecross/regenerative/pyrite
 	colour = "pyrite"
+	effect_desc = "Fully heals the target and randomly colors the target."
 
 /obj/item/slimecross/regenerative/pyrite/core_effect(mob/living/target, mob/user)
 	target.visible_message("<span class='warning'>The milky goo coating [target] leaves [target.p_them()] a different color!</span>")
 	target.add_atom_colour(rgb(rand(0,255),rand(0,255),rand(0,255)),WASHABLE_COLOUR_PRIORITY)
 
 /obj/item/slimecross/regenerative/red
 	colour = "red"
+	effect_desc = "Fully heals the target and injects the target with some ephedrine."
```

Comment:
Fully heals the target and injects them with some ephedrine.

I don't like having to use "them" but we often have to.